scripts/smoketest.py

```diff
@@ -1,56 +1,114 @@
 import os
+import tempfile
 import time
 from pathlib import Path
 
 import pytest
 import requests
+from unstructured.partition.auto import partition
+from unstructured.staging.base import convert_to_csv
 
 API_URL = "http://localhost:8000/general/v0/general"
 # NOTE(rniko): Skip inference tests if we're running on an emulated architecture
 skip_inference_tests = os.getenv("SKIP_INFERENCE_TESTS", "").lower() in {"true", "yes", "y", "1"}
 
 
-def send_document(filename, content_type, strategy="fast"):
+def send_document(filename, content_type, strategy="fast", output_format="application/json"):
     files = {"files": (str(filename), open(filename, "rb"))}
-    return requests.post(API_URL, files=files, data={"strategy": strategy})
+    return requests.post(API_URL, files=files, data={"strategy": strategy, "output_format": output_format})
 
 
+# NOTE(kravetsmic): This function was added for removing path from text/csv response because the backend has a different
+# temporary path than the temporary path in tests
```

Comment:
the backend server should never be sending back a temporary path (unless the client actually posted that as the filename in its form submission as the filename)

Reply:
so, there is another bug that needs to be resolved here, not related to this PR

Reply:
i believe the bug that prompted the workaround here has now been resolved in main, so hopefully this `remove_path` function may be removed.

Reply:
@kravetsmic  , is this note still valid? any temporary paths should have been removed with https://github.com/Unstructured-IO/unstructured-api/commit/d876a86195ed94e7fcd1d5b53662ab5aa15d27e1#diff-d80162975e0983c4067dc93f0d9a06f8867a7a982cf67a38975109e448d20c5e